lib/agent0/agent0/hyperdrive/interactive/interactive_hyperdrive_test.py

```diff
@@ -590,3 +619,55 @@ def test_policy_config_forgotten(chain: LocalChain):
         policy=Zoo.random,
     )
     assert alice.agent.policy is not None
+
+
+@pytest.mark.anvil
+def test_share_price_quincunx(chain: Chain):
+    """Share price when compounding by quincunx."""
+    # setup
+    initial_variable_rate = FixedPoint("0.045")
+    interactive_config = InteractiveHyperdrive.Config(
+        position_duration=YEAR_IN_SECONDS,  # 1 year term
+        governance_lp_fee=FixedPoint(0),
+        curve_fee=FixedPoint(0),
+        flat_fee=FixedPoint(0),
+        initial_variable_rate=initial_variable_rate,
+    )
+    interactive_hyperdrive = InteractiveHyperdrive(chain, interactive_config)
+    hyperdrive_interface = interactive_hyperdrive.hyperdrive_interface
+    arbitrage_andy = create_arbitrage_andy(interactive_hyperdrive=interactive_hyperdrive)
+    logging.info(f"Variable rate: {hyperdrive_interface.current_pool_state.variable_rate}")
+    logging.info(f"Starting share price: {hyperdrive_interface.current_pool_state.pool_info.lp_share_price}")
+    number_of_compounding_periods = 5
+    for _ in range(number_of_compounding_periods):
+        chain.advance_time(YEAR_IN_SECONDS // number_of_compounding_periods, create_checkpoints=False)
+        arbitrage_andy.open_long(FixedPoint(20))
```

Comment:
can you add a comment here that says your opening a trade in order to trigger interest accumulation?

Reply:
or try `interactive_hyperdrive.interface.create_checkpoint(...)` instead of open long

Reply:
even if you do a checkpoint you should add a comment